src/EditorFeatures/CSharpTest/Interactive/NavigateTo/InteractiveNavigateToTests.cs

```diff
@@ -295,7 +287,7 @@ class Goo
 }", async w =>
             {
                 var item = (await _aggregator.GetItemsAsync("DS")).Single();
-                VerifyNavigateToResultItem(item, "DoSomething", "[|D|]o[|S|]omething()", MatchKind.Regular, NavigateToItemKind.Method, Glyph.MethodPrivate, string.Format(FeaturesResources.in_0_project_1, "Goo", "Test"));
+                VerifyNavigateToResultItem(item, "DoSomething", "[|D|]o[|S|]omething()", PatternMatchKind.CamelCaseExact, NavigateToItemKind.Method, Glyph.MethodPrivate, string.Format(FeaturesResources.in_0_project_1, "Goo", "Test"));
```

Comment:
can you confirm you've gone over all the differences and have validated that teh new value is appropriate?  (in this case it is, but i want to know if i should go review each change, or if that's already been done).  Thanks!

Reply:
I have confirmed all of them. Other than patterns with underscores, all tests are returning appropriate values.